source/iostreams.tex

```diff
@@ -7756,17 +7762,33 @@
 void vprint_unicode(FILE* stream, string_view fmt, format_args args);
 \end{itemdecl}
 
+\begin{itemdescr}
+\pnum
+\effects
+Equivalent to:
+\begin{codeblock}
+string out = vformat(fmt, args);
+vprint_unicode_locking(stream, "{}", make_format_args(out));
+\end{codeblock}
+\end{itemdescr}
+
+\indexlibraryglobal{vprint_unicode_locking}%
+\begin{itemdecl}
+void vprint_unicode_locking(FILE* stream, string_view fmt, format_args args);
+\end{itemdecl}
+
 \begin{itemdescr}
 \pnum
 \expects
 \tcode{stream} is a valid pointer to an output C stream.
 
 \pnum
 \effects
-The function initializes an automatic variable via
-\begin{codeblock}
-string out = vformat(fmt, args);
-\end{codeblock}
+Locks \tcode{stream}.
+See also \IsoC{}, 7.21.2.
+Let \tcode{out} denote the character representation of
+formatting arguments provided by \tcode{args}
+formatted according to specifications given in \tcode{fmt}.
```

Comment:
This is what the incoming paper says, but I wonder if "according to the specifications" would be better.